### PR TITLE
Add about us page content

### LINKED ISSUE #13: Add about us page content

### LINKED ISSUE #32: Add contact page content
Connect "Csatlakozz!" button on front page to this page.

### PR DESCRIPTION
**Pull request by triszt4n**
Closes #13 
Closes #32

### CODE REVIEW

File: src/components/about-components/StickyNote.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { Box, theme, useColorModeValue } from '@chakra-ui/react'
+import React from 'react'
+
+const StickyNote: React.FC = ({ children }) => (
+  <Box
+    bgGradient="linear(to-br, yellow.100, orange.300)"
```
Comment (OmTheTurtle) [inline src/components/about-components/StickyNote.tsx:6]: `orange.300` feels a bit too dark together with the light font weigtht, maybe try `orange.200`

File: src/pages/about.tsx [hunk 2]
```diff
@@ -1,82 +1,342 @@
...
+                    height: useBreakpointValue({ base: '4rem', md: '6rem' })
+                  }}
+                />
+              </Box>
+              <AboutParagraph title="Történelem" titleSize={secondHeadingSize}>
+                <Text mt={6}>
+                  A Kollégiumi Információs Rendszer Fejlesztői és Üzemeltetői – röviden Kir-Dev – a BME VIK hallgatóiból álló webfejlesztő
+                  csapat. Körünk 2001-ben alakult, 2009 óta a{' '}
+                  <Link _hover={{ textDecor: 'none' }} href="https://simonyi.bme.hu/">
+                    <Text display="inline" w="full" bgClip="text" bgGradient="linear(to-r, green.400, teal.500)" fontWeight="extrabold">
```
Comment (OmTheTurtle) [inline src/pages/about.tsx:65]: I'd use a solid color inside a paragraph instead of the gradient.

File: src/pages/about.tsx [hunk 3]
```diff
@@ -1,82 +1,342 @@
...
+                      Simonyi Károly Szakkollégium
+                    </Text>
+                  </Link>{' '}
+                  része. Alakulásunk óta foglalkozunk különféle webes technológiák alkalmazásával és oktatásával, valamint fejlesztünk és
+                  üzemeltetünk a kollégiumi közösség számára hasznos webes alkalmazásokat.
+                </Text>
+                <Text py={2}>
```
Comment (OmTheTurtle) [inline src/pages/about.tsx:72]: I'm not sure if it's intentional, but on the deploy preview this paragraph gets a gradient text color, which I personally don't really like. If it's a bug it should be investigated.

image

File: src/pages/about.tsx [hunk 4]
```diff
@@ -1,82 +1,342 @@
...
+                  Fő feladatunk a{' '}
+                  <GatsbyLink to="/project/pek-next">
+                    <Text display="inline" w="full" bgClip="text" bgGradient="linear(to-r, tomato, orange.300)" fontWeight="extrabold">
```
Comment (OmTheTurtle) [inline src/pages/about.tsx:75]: No gradients, use solid color.

File: src/pages/about.tsx [hunk 5]
```diff
@@ -1,82 +1,342 @@
...
+              </Text>
+            </AboutParagraph>
+            <AboutParagraph title="Kommunikáció" id="contact" titleSize={secondHeadingSize}>
+              <Flex mt={6} direction={{ base: 'column', sm: 'row' }}>
+                <Text pr={{ base: 0, sm: 10 }}>
+                  Ha maradt még kérdésed, hibajegyet szeretnél feladni valamilyen projektünkkel kapcsolatban, vagy van valami izgalmas
+                  feladatod számunkra: vedd fel a kapcsolatot velünk email címünkön, esetleg gyere el gyűléseinkre a Schönherz Kollégium
+                  1319-es szobájába!
+                </Text>
+                <Flex flex={1} whiteSpace="nowrap" width="full" direction="column" alignItems="flex-end">
```
Comment (OmTheTurtle) [inline src/pages/about.tsx:311]: The vertical spacing here should be larger between the items.

File: src/pages/about.tsx [hunk 6]
```diff
@@ -1,82 +1,342 @@
...
-          <Link id="contact" href="." />
-          <Heading as="h1">Kapcsolat</Heading>
+          <Box textAlign="center" my={6}>
+            <Heading mb={6} fontSize={{ base: '4xl', sm: '5xl', md: '6xl' }} fontWeight="bold" lineHeight="none" letterSpacing="tight">
+              A{' '}
+              <Text display="inline" w="full" bgClip="text" bgGradient="linear(to-r, tomato, orange.300)" fontWeight="extrabold">
+                Schönherz koli
+              </Text>{' '}
+              webfejlesztői
+            </Heading>
```
Comment (OmTheTurtle) [inline src/pages/about.tsx:47]: As discussed, the text should read `A Schönherz webfejlesztő köre` with the gradient applied to the word 'webfejlesztő'

File: src/pages/about.tsx (hunk 3 above)
Comment (triszt4n) [reply src/pages/about.tsx:72]: Wow, that's weird

File: src/pages/about.tsx (hunk 6 above)
Comment (triszt4n) [reply src/pages/about.tsx:47]: Thanks, completely forgot what the text should've been :sweat_smile:

Comment (triszt4n) [reply src/pages/about.tsx]: I've used `A kollégium webfejlesztő köre`, because Schönherz with its capital letter was weird in as the title

**Review by OmTheTurtle — APPROVED** [review]
We should do something to separate the content from the layout, it's not really optimal to have the two in the same huge file. Maybe we can use MDX so we still can format it how we want while the content itself is more decoupled.

**Comment by triszt4n** [comment]
I agree, but another big plugin would further slow down the build times. :thinking: I'll see if there are smart solutions for that, and if nothing else remains, I still can separate the file into smaller components (like the techstack component,  etc.) and work on more modularized styles.

**Comment by triszt4n** [comment]
Alright I guess this would mean more overhead for the AboutPage to be separated into components, and the MDX would need nasty styles or a common scss file for them. I dunno, maybe it's good as it is, and we should make an issue for the future. (Just in case, if the about page gets more and more content) If it's alright, I'll merge this, once @SepsiLaszlo and @poiuztrewq1 approves of the content (mostly the text).

**Review by poiuztrewq1 — APPROVED** [review]

File: src/pages/about.tsx [hunk 7]
```diff
@@ -1,81 +1,339 @@
...
+                  Fő feladatunk a{' '}
+                  <GatsbyLink to="/project/pek-next">
+                    <chakra.span color={linkColor} _hover={linkHover}>
+                      Profil és Körök
+                    </chakra.span>
+                  </GatsbyLink>{' '}
+                  – röviden PéK – folyamatos fejlesztése és karbantartása. Jelenleg ezen az alkalmazáson keresztül folyik a kar közösségi
+                  pontozása. Ez a rendszer már több generációt is megélt az aktív körtagoknak köszönhetően: legelőször Kollégiumi
+                  Információs Rendszernek hívtuk (KIR), aztán a kiterjedésével Villanykari Információs Rendszer (VIR) lett, később pedig
+                  megkapta modern nevét, a PéK-et. A korábbi generációi még JavaEE-s alapokon, OpenAM mellett futottak, aztán 2015-ben Ruby
```
Comment (salierri) [inline src/pages/about.tsx:79]: Kis pontosítás: az eredeti KIR még PHP-ban íródott, amikor újraírták JavaEE-ben, akkor lett belőle VIR

Comment (triszt4n) [reply src/pages/about.tsx]: Ki is javítottam, köszi!

**Review by SepsiLaszlo — APPROVED** [review]

**Review by salierri — APPROVED** [review]

**Pull request merged**